redwood-treehouse-host/src/iosTest/kotlin/app/cash/redwood/treehouse/TreehouseUIViewTest.kt

```diff
@@ -128,4 +131,21 @@ class TreehouseUIViewTest {
     assertThat(layout.uiConfiguration.value)
       .isEqualTo(UiConfiguration(safeAreaInsets = expectedInsets))
   }
+
+  @Test
+  fun uiConfigurationReflectsInitialLayoutDirection() {
+    val parent = UIWindow()
+
+    val layout = TreehouseUIView(throwingWidgetSystem)
+    parent.addSubview(layout.view)
+
+    val expectedLayoutDirection = when (val direction = parent.effectiveUserInterfaceLayoutDirection) {
+      UIUserInterfaceLayoutDirectionLeftToRight -> LayoutDirection.Ltr
+      UIUserInterfaceLayoutDirectionRightToLeft -> LayoutDirection.Rtl
```

Comment:
I didn't realize iOS had this. We have some missing tests for the layout, it seems! https://github.com/cashapp/redwood/issues/2092